cloudify_gcp/compute/network.py

```diff
@@ -226,8 +229,10 @@ def add_peering(name, network, peerNetwork, autoCreateRoutes, **kwargs):
             network=network,
             peerNetwork=peerNetwork,
             autoCreateRoutes=autoCreateRoutes)
-
-    utils.create(peer)
+    try:
+        utils.create(peer)
+    except HttpError as e:
+        raise RecoverableError(e.message)
     ctx.instance.runtime_properties[constants.RESOURCE_ID] = peer.name
```

Comment:
Why should HTTP Error raise recoverable error? It seems like other uses of `utils.create()` below do not happen in try/except.

Reply:
there is some error got from gcp when creating peering, the error says try again:

```
There is a peering operation in progress on the local or peer network. Try again later.
```
The type we got for this error is `HTTPError`, this is why I'm raising recoverable error... 

Sorry what you mean by other uses?